gamemodes/jazztronauts/entities/entities/jazz_bus_explore/init.lua

```diff
@@ -215,6 +215,11 @@ function ENT:AttachRadio(pos, ang)
 
 	-- Attach a looping audiozone
 	self.RadioMusic = CreateSound(ent, self.RadioMusicName)
+	hook.Add("EntityRemoved", "CheckTheRadio", function(removed)
```

Comment:
Really minor, and I realized I should've written this on the previous PRs, but for gmod hooks I try to use a a "jazz" prefix to keep it unique. I wouldn't be surprised if there's some other rascal out there with a radio mod and a hook called CheckTheRadio.

Reply:
But what if they have a jazz radio mod?

Reply:
Ah, you're absolutely right! Thankfully the others are fairly specific, only quickly running and with a rather specific name and hook, but this one is definitely far too vague. I've changed it to "JazzBusRadioCheck", which I don't think anything will conflict with. If we find an addon with a jazzy bus that lets you check the radio, then we'll deal with it then ;D